.github/workflows/actions.yml

```diff
@@ -0,0 +1,65 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: JUnit Pioneer Flow
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: '*'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # test against latest update of each major Java version, as well as specific updates of LTS versions:
+        java: [ 8, 11 ]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    name: ${{ matrix.os }} - Java ${{ matrix.java }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Gradle build
+        uses: eskatos/gradle-command-action@v1
```

Comment:
Why use an external action here? We already have the Gradle Wrapper in-place and ready to rock.

Reply:
i thought actually to be the save way, as it also uses the gradle wrapper, but i do not need to care about write permissions etc. - i saw that in junit5 and i found it odd. i did not test it without it. but this is something which we definitly can investigate

Reply:
ah damn, that is something i have seen somewerhe else sorry for wrong reference to junit5

Reply:
also there is https://github.com/eskatos/gradle-github-actions-plugin which might be useful in the future

Reply:
![image](https://user-images.githubusercontent.com/9987394/77700115-d87b2680-6fb3-11ea-8cc2-cae56fb824ba.png)
no need to specify the gradle wrapper as the gradlew is still in the root directory

Reply:
Okay. Let's use `eskatos/gradle-command-action` and see what it brings to the party. Might consider using it for JUnit 5 as well.

/cc @marcphilipp @eskatos